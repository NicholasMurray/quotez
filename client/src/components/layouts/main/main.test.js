import React from 'react';
import TestRenderer from 'react-test-renderer';
import 'jest-styled-components';
import Main from '.';

test('it renders the main ui element correctly', () => {
  const tree = TestRenderer.create(<Main />).toJSON();
  expect(tree).toMatchSnapshot();
});
