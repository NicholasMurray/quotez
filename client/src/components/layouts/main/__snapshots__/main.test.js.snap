// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the main ui element correctly 1`] = `
.c0 {
  text-align: center;
  margin: 1rem 4rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<main
  className="c0"
/>
`;
